admin/src/utils.js

```diff
@@ -156,6 +156,26 @@ export const translate = (value) => {
       return "Mère";
     case "representant":
       return "Représentant légal";
+    case "SERVER_ERROR":
+      return "Erreur serveur";
+    case "NOT_FOUND":
+      return "Existe pas";
```

Comment:
ça fait bizarre peut être : 

```suggestion
      return "Ressource introuvable";
```